### DISCUSSION TITLE
Check exit code

### DISCUSSION THREAD

**Discussion by fellnerse** [nushell/nushell#4233] [Q&A] [answered]
Hello,

small question: how do I check the exit code of an external program?
i.e. `let a = (foo bar)`
and now I would like to know what the exit code of this is. Could not find anything in issues, or in the cookbook

**Comment by kubouch** [comment, marked as answer]
We do not support it in Nushell.

It's something we'd like to have in the new engine but I think it turned out to be surprisingly tricky to implement. I haven't worked on it so I can't tell the exact status.

**Reply by fellnerse** [reply]
Ok thanks
